src/sas/qtgui/Utilities/GridPanel.py

```diff
@@ -236,19 +234,19 @@ def actionSendToExcel(self):
             self.writeBatchToFile(data=data, tmpfile=tmpfile, details=details)
             tmpfile.close()
 
-        try:
-            from win32com.client import Dispatch
-            excel_app = Dispatch('Excel.Application')
-            excel_app.Workbooks.Open(self.grid_filename)
-            excel_app.Visible = 1
-        except Exception as ex:
-            msg = "Error occured when calling Excel.\n"
-            msg += ex
-            self.parent.communicate.statusBarUpdateSignal.emit(msg)
+        mime_type = QMimeDatabase().mimeTypeForFile(self.grid_filename)
+
+        if mime_type.isValid():
+            url = QUrl.fromLocalFile(self.grid_filename)
+
+            if QDesktopServices.openUrl(url):
+                print("File opened successfully.")
+            else:
+                print("Failed to open file.")
```

Comment:
`print` statements will not be seen in the GUI. Please replace them with `self.parent.communicate.statusBarUpdateSignal.emit(<msg>)` which will send the desired message to the console log.

Suggested wording:

- Success: `The batch results csv file successfully opened in your system csv viewer.`
- Failure: `A csv viewer is required to view the batch results. Please set one in your default app settings to change this behavior.`